framework-test/build.gradle

```diff
@@ -12,6 +12,8 @@ dependencies {
     implementation project(':javacutil')
     implementation project(':checker-qual')
 
+    implementation 'org.plumelib:plume-util:1.1.5'
```

Comment:
This dependency needs to be added to `checker-framework/docs/developer/release/maven-artifacts/framework-test-pom.xml`.